features/invite/api/src/main/kotlin/io/element/android/features/invite/api/response/AcceptDeclineInviteEvents.kt

```diff
@@ -8,6 +8,6 @@
 package io.element.android.features.invite.api.response
 
 interface AcceptDeclineInviteEvents {
-    data class AcceptInvite(val invite: InviteData) : AcceptDeclineInviteEvents
-    data class DeclineInvite(val invite: InviteData) : AcceptDeclineInviteEvents
+    data class AcceptInvite(val invite: InviteData?) : AcceptDeclineInviteEvents
+    data class DeclineInvite(val invite: InviteData?, val blockUser: Boolean = false) : AcceptDeclineInviteEvents
```

Comment:
Weird to see optional for `InviteData`, see my other comment

Reply:
Yeah I do agree, but otherwise we'll have to manage null case error in several places... so not sure what is best

Reply:
Keep it like that then, it's fine